### PR TITLE
sc: add page

### PR DESCRIPTION
**Pull request by FallenGameR**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/windows/sc.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# sc
+
+> Communicate with the Service Control Manager and services on Windows from the command-line.
+> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/sc-query>.
+
+- Show status of a service:
```
Comment (kbdharun) [inline pages/windows/sc.md:6]:
```suggestion
- Show the status of a service:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a minor suggestion for this page.

File: pages/windows/sc.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# sc
+
+> Communicate with the Service Control Manager and services on Windows from the command-line.
```
Comment (acuteenvy) [inline pages/windows/sc.md:3]:
```suggestion
> Communicate with the Service Control Manager and services.
```

Comment (kbdharun) [reply pages/windows/sc.md:3]: Regarding this, we might need to fix it in other Windows pages too.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**